chain/beacon/drand/drand.go

```diff
@@ -84,6 +89,7 @@ func NewDrandBeacon(genesisTs, interval uint64, ps *pubsub.PubSub, config dtypes
 		if err != nil {
 			return nil, xerrors.Errorf("could not create http drand client: %w", err)
 		}
+		hc.(DrandHTTPClient).SetUserAgent("drand-client-lotus/" + build.BuildVersion)
```

Comment:
Would it be possible to discriminate between miner/node etc. ?

Also, `CurrentCommit` might be better than BuildVersion (or both).

Reply:
i was worried that the distinct commit was a bit of a privacy invasive thing. There isn't a build constant for type of node, and i didn't see an easy way to ask lotus for that.

Reply:
the `os.Executable` thing that would get used by default would give that right?

Reply:
maybe, but not reliably.
if the thing is being run via a `go run` then that doesn't work. Better to say lotus here, i think

Reply:
Not that anyone is ever going to run lotus via `go run` I hope. But ok, eventually only the miners should be querying drand anyways.

Reply:
Miner process always uses lotus daemon for getting its randomess.